### PR TITLE
Java Edition 1.21.2 / 1.21.3 support

### PR DESCRIPTION
**Pull request by Camotoy**
See here for a general idea of what needs to be addressed: https://github.com/GeyserMC/Geyser/issues/5100

### CODE REVIEW

File: build-logic/src/main/kotlin/geyser.modrinth-uploading-conventions.gradle.kts [hunk 1]
```diff
@@ -12,7 +12,7 @@ modrinth {
     versionNumber.set(projectVersion(project))
     versionType.set("beta")
     changelog.set(System.getenv("CHANGELOG") ?: "")
-    gameVersions.addAll("1.21", libs.minecraft.get().version as String)
+    gameVersions.addAll("1.21.2", libs.minecraft.get().version as String)
```
Comment (Camotoy) [inline build-logic/src/main/kotlin/geyser.modrinth-uploading-conventions.gradle.kts:15]: 1.21.3 should be added.

Comment (onebeastchris) [reply build-logic/src/main/kotlin/geyser.modrinth-uploading-conventions.gradle.kts:15]: that's already being done by the libs' Minecraft version reference

Comment (Camotoy) [reply build-logic/src/main/kotlin/geyser.modrinth-uploading-conventions.gradle.kts:15]: Project maintainer reading comprehension challenge FAILED

**Comment by dom111** [comment]
Hey @Camotoy,

Thanks for your hard work on this, great to see the progress happen!

I've been testing this PR branch on my server, apologies if this is not the right place to ask, but would you like any issues we encounter to be logged somewhere (probably not yet, but when you're working towards an RC maybe?)

Thanks!

Dom

**Comment by Camotoy** [comment]
Hi! We are nearing a stable build. Until we do, you are free to log any issues you find here.

**Pull request merged**